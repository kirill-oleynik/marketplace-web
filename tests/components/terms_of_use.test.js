const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const { TermsOfUse } = require('../../app/components/terms_of_use');

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <TermsOfUse
        isOpen
        toggle={() => {}}
        t={(translation) => translation}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
