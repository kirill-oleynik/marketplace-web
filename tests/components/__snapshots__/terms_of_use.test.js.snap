// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<Modal
  autoFocus={true}
  backdrop={false}
  backdropTransitionTimeout={150}
  className="main-modal"
  fade={true}
  isOpen={true}
  keyboard={true}
  modalTransitionTimeout={300}
  toggle={[Function]}
  zIndex={1050}
>
  <Container
    tag="div"
  >
    <div
      className="text-right pt-30"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        <i
          className="icon icon-cross font-16 in-black-035"
        />
      </button>
    </div>
    <div
      className="main-modal__content pt-20 pb-20"
    >
      <h1
        className="font-24 font-700 mb-30"
      >
        termsOfUse.title
      </h1>
      <h2
        className="font-16 font-700 mb-20"
      >
        termsOfUse.welcome
      </h2>
      <p
        className="in-black-050 mb-30"
      >
        termsOfUse.accessing
      </p>
      <h3
        className="font-16 mb-20"
      >
        termsOfUse.usage.title
      </h3>
      <ul
        className="in-black-050 mb-30"
      >
        <li
          className="main-list-item mb-5"
        >
          termsOfUse.usage.services
        </li>
        <li
          className="main-list-item mb-5"
        >
          termsOfUse.usage.content
        </li>
        <li
          className="main-list-item"
        >
          termsOfUse.usage.individuals
        </li>
      </ul>
      <h3
        className="font-16 mb-20"
      >
        termsOfUse.definitions.title
      </h3>
      <ul
        className="in-black-050 mb-30"
      >
        <li
          className="main-list-item mb-5"
        >
          termsOfUse.definitions.services
        </li>
        <li
          className="main-list-item mb-5"
        >
          termsOfUse.definitions.content
        </li>
        <li
          className="main-list-item mb-5"
        >
          termsOfUse.definitions.individuals
        </li>
        <li
          className="main-list-item mb-5"
        >
          termsOfUse.definitions.users
        </li>
        <li
          className="main-list-item mb-5"
        >
          termsOfUse.definitions.location
        </li>
      </ul>
      <h3
        className="font-16 mb-20"
      >
        termsOfUse.rights.title
      </h3>
      <ol
        className="main-list-numbered in-black-050 mb-30"
      >
        <li
          className="main-list-numbered__item mb-5"
        >
          termsOfUse.rights.services
        </li>
        <li
          className="main-list-numbered__item mb-5"
        >
          termsOfUse.rights.content
        </li>
        <li
          className="main-list-numbered__item"
        >
          termsOfUse.rights.individuals
        </li>
      </ol>
      <p
        className="font-700 mb-20"
      >
        termsOfUse.agree
      </p>
      <p
        className="font-700"
      >
        termsOfUse.lastUpdate
      </p>
    </div>
  </Container>
</Modal>
`;
